tfhe/src/high_level_api/glwe_ct_list.rs

```diff
@@ -0,0 +1,245 @@
+use super::global_state::with_cpu_internal_keys;
+use crate::core_crypto::commons::math::random::{Deserialize, Serialize};
+use crate::high_level_api::integers::{FheIntId, FheUintId};
+use crate::integer::ciphertext::{DataKind, Expandable, GlwePackable};
+use crate::named::Named;
+use crate::shortint::{Ciphertext, MessageModulus};
+use crate::{FheBool, FheInt, FheUint};
+
+impl<Id: FheUintId> GlwePackable for FheUint<Id> {
+    fn compact_into(
+        self,
+        messages: &mut Vec<Ciphertext>,
+        message_modulus: MessageModulus,
+        num_blocks: Option<usize>,
+    ) -> DataKind {
+        match self.ciphertext {
+            super::integers::unsigned::inner::RadixCiphertext::Cpu(a) => {
+                a.compact_into(messages, message_modulus, num_blocks)
+            }
+            #[cfg(feature = "gpu")]
+            super::integers::unsigned::inner::RadixCiphertext::Cuda(_) => panic!(),
+        }
+    }
+}
+
+impl<Id: FheIntId> GlwePackable for FheInt<Id> {
+    fn compact_into(
+        self,
+        messages: &mut Vec<Ciphertext>,
+        message_modulus: MessageModulus,
+        num_blocks: Option<usize>,
+    ) -> DataKind {
```

Comment:
doesn't feel like that useful given the Radix implem gets that info on its own

Reply:
Removed